### PR TITLE
Fix crash when double click to open `FileHistory` form

### LINKED ISSUE #4524: NRE when opening `File history` form (regression in `master`)
**Do you want to request a *feature* or report a *bug*?**
bug (regression in `master`)

**What is the current behavior?**

When double click on a file in the `Diff` or `File tree` tab of the browse form, a `System.NullReferenceException` occurs instead of opening the `File history` form.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**

Open the `File history` form.

**Environment you encounter the issue:**

- GitExtensions version: developpment version 
- GIT version: 2.16
- OS version: 10

**Did this work in previous version of GitExtensions (which)?**
yes, 2.51

### PR DESCRIPTION
**Pull request by pmiossec**
from 'file tree' and 'diff' tabs

Fixes #4524

### CODE REVIEW

File: GitUI/CommandsDialogs/RevisionDiff.cs [hunk 1]
```diff
@@ -313,8 +313,7 @@ private void DiffFiles_DoubleClick(object sender, EventArgs e)
             }
             else
             {
-
-                UICommands.StartFileHistoryDialog(this, (DiffFiles.SelectedItem).Name);
+                OpenFileHistoryDialogForItem((DiffFiles.SelectedItem).Name);
```
Comment (RussKie) [inline GitUI/CommandsDialogs/RevisionDiff.cs]: Redundant brackets

Comment (pmiossec) [reply GitUI/CommandsDialogs/RevisionDiff.cs]: There was brackets somewhere !?! 😄

**Comment by gerhardol** [comment]
With #4528 merged, I propose the changes in RevisionDiff is closed. RevisionFileTree.cs is relevant.

**Comment by RussKie** [comment]
@pmiossec is your fix still relevant?

**Comment by pmiossec** [comment]
I have to verify (and test) but I think it still is...

**Comment by gerhardol** [comment]
RevisionDiff.cs should use DiffFiles.Revision instead, but that is not fully initialized. 
(I believe) the first occurrence of will currently open HEAD and not the selected revision but would prefer to not have yet another layer here.

**Comment by pmiossec** [comment]
Thanks @gerhardol, I fixed that.

@RussKie I have tested and yes it is still relevant. With this PR, it will select the current selected revision in the file history instead of selecting the last revision of the file.

**Comment by RussKie** [comment]
Awesome, thanks

**Pull request merged**